### DISCUSSION TITLE
How to use admonition syntax boxings on GitHub READMEs?

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#37] [Q&A] [answered]
How to add admonition boxes for user's attraction on GitHub READMEs?

**Comment by andatoshiki** [comment, marked as answer]
While not exactly an admonition, this is supported (beta tested) since May 2022, as [[reported by Diego Haz](https://twitter.com/diegohaz/status/1527642881384759297)](https://twitter.com/diegohaz/status/1527642881384759297) and described in [[discussion 16925](https://github.com/github/feedback/discussions/16925)](https://github.com/github/feedback/discussions/16925)

> To better highlight and separate certain information from the rest in your documentation on GitHub, we now render a special and accessible note or warning blockquote in Markdown documents.
>
> We are using the existing syntax for blockquote and bold text.
>
> The first line must be exactly as shown below.
> The first letter is case sensitive. The second line can contain your content.
>
> This input:
>
> ```
> > **Note**
> > This is a note
> 
> > **Warning**
> > This is a warning
> ```
>
> Becomes:
>
> [![Note / This is a note and Warning / This is a warning](https://i.stack.imgur.com/FiB6U.png)](https://i.stack.imgur.com/FiB6U.png)

Real example as follows:
> **Note**
> This is a note

> **Warning**
> This is a warning

Warning: the syntax might still evolve and is debated in the discussion. For instance, [[Brenton M. Wiernik](https://github.com/bwiernik)](https://github.com/bwiernik) [[comments](https://github.com/github/feedback/discussions/16925#discussioncomment-2795312)](https://github.com/github/feedback/discussions/16925#discussioncomment-2795312):

> In addition to syntax issues and semantic web problems raised by others, I am also concerned about this introducing a major incompatibility between GitHub-Flavored Markdown and pandoc markdown or CommonMark.
>
> A large number of R developers write READMEs and other documentation in RMarkdown, which is generally processed using pandoc.
>
> This change would mean that users would be unable to get consistent rendering with the same syntax when their package documentation is displayed on GitHub vs elsewhere.
>
> Overloading the blockquote in this way creates an opaque difference in the meaning of syntax across markdown flavors that is very difficult to write around for developers.
